controller/admin.go

```diff
@@ -188,6 +199,187 @@ func AdminAssignmentGET(w http.ResponseWriter, r *http.Request) {
 	v.Render(w)
 }
 
+// AdminAssignmentCreateGET handles GET-request from /admin/assigment/create
+func AdminAssignmentCreateGET(w http.ResponseWriter, r *http.Request) {
+	//check that user is a teacher
+	if !session.IsTeacher(r) { //not a teacher, error 401
+		ErrorHandler(w, r, http.StatusUnauthorized)
+		return
+	}
+
+	var subRepo = model.SubmissionRepository{}
+	submissions, err := subRepo.GetAll()
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+
+	v := view.New(r)
+	v.Name = "admin/assignment/create"
+
+	v.Vars["Courses"] = []struct { // TODO (Svein): use database
+		ID   int
+		Code string
+		Name string
+	}{
+		{ID: 1, Code: "IMT1031", Name: "Grunnleggende Programmering"},
+		{ID: 2, Code: "IMT1082", Name: "Objekt-Orientert Programmering"},
+		{ID: 3, Code: "IMT2021", Name: "Algoritmiske Metoder"},
+	}
+
+	v.Vars["Submissions"] = submissions
+
+	v.Render(w)
+}
+
+// AdminAssignmentCreatePOST handles POST-request from /admin/assigment/create
+func AdminAssignmentCreatePOST(w http.ResponseWriter, r *http.Request) {
+	//check that user is a teacher
+	if !session.IsTeacher(r) { //not a teacher, error 401
+		ErrorHandler(w, r, http.StatusUnauthorized)
+		return
+	}
+
+	assignmentRepository := model.AssignmentRepository{}
+
+	publish, err := DatetimeLocalToRFC3339(r.FormValue("publish"))
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
+	deadline, err := DatetimeLocalToRFC3339(r.FormValue("deadline"))
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
```

Comment:
Something like: 
```suggestion
if deadline > publish{
    log.println("error")
    return
}
```
Don't let deadline be before publish date

Reply:
also use ErrorHandler() more so end user can see somethings wrong